plugin/src/test/java/org/opensearch/ml/action/config/GetConfigTransportActionTests.java

```diff
@@ -162,9 +162,32 @@ public void testDoExecute_Success() throws IOException {
         verify(actionListener).onResponse(any(MLConfigGetResponse.class));
     }
 
+    @Test
+    public void testDoExecute_Success_ForNewFields() throws IOException {
+        String configID = "config_id";
+        MLConfig mlConfig = new MLConfig(null, "olly_agent", null, new Configuration("agent_id"), Instant.EPOCH, null, Instant.EPOCH);
```

Comment:
can you explain this test why one of the configType, configuration and mlConfiguration has value 'olly_agent' and the rest of the two are null? 

```
            .configType(configType)
            .configuration(configuration)
            .mlConfiguration(mlConfiguration)
```